message/validation/signed_ssv_message.go

```diff
@@ -67,6 +67,11 @@ func (mv *messageValidator) validateSignedSSVMessage(signedSSVMessage *spectypes
 			return ErrDuplicatedSigner
 		}
 		prevSigner = signer
+
+		// Rule: Reject messages from removed operators
+		if err := mv.validateOperator(signer); err != nil {
+			return err
+		}
 	}
 
 	// Rule: Len(Signers) must be equal to Len(Signatures)
```

Comment:
lets make sure we don't assign `OperatorIDs` as the whole committee if an operator was removed. (if we do this rule will fail)

Reply:
i filtered out removed operators, let me know if it looks good now, these rules are also covered in tests